server/src/reactutils.js

```diff
@@ -1,12 +1,26 @@
 exports.addReactScripts = function (body, addScript) {
-  let footer = "</body></html>";
-  let frontmatter = body.slice(0, body.length - footer.length);
-  addScript = addScript.replace(/<script/ig, "\\x3cscript").replace(/<\/script/ig, "\\x3c/script");
-  return (`<!DOCTYPE html>
-${frontmatter}
-<script>
-${addScript}
-</script>
-${footer}`
-  );
+  if (addScript) {
+    let footer = "</body></html>";
+    let frontmatter = body.slice(0, body.length - footer.length);
+    addScript = addScript.replace(/<script/ig, "\\x3cscript").replace(/<\/script/ig, "\\x3c/script");
+    return (`<!DOCTYPE html>
+  ${frontmatter}
+  <script>
+  ${addScript}
+  </script>
+  ${footer}`
+    );
+  } else {
+    return `<!DOCTYPE html>\n${body}`;
+  }
+};
+
+exports.extend = function (...objs) {
+  let obj = {};
```

Comment:
It is implemented in a polyfill in babel, core.js

    https://github.com/zloirock/core-js#ecmascript-6-object